src/components/UserCard/index.jsx

```diff
@@ -95,9 +235,15 @@ User.propTypes = {
   username: string.isRequired,
   roles: object,
   permissions: object,
+  scheduledPermissions: object,
+  onSignoff: func.isRequired,
+  onRevoke: func.isRequired,
 };
 
 User.defaultProps = {
   roles: {},
   permissions: {},
+  scheduledPermissions: {},
 };
+
+export default withUser(User);
```

Comment:
I think adding `marginBottom: theme.spacing(4)` to the Grid container will make it easier to distinguish the change from the required signoffs:

![Screen Shot 2019-07-30 at 9 42 02 AM](https://user-images.githubusercontent.com/3766511/62134423-73e5bb80-b2ae-11e9-99d5-377a54265ac3.png)